### DISCUSSION TITLE
How to use SSR with SUID?

### DISCUSSION THREAD

**Discussion by awojciak** [swordev/suid#111] [Q&A]
Hi, I'm getting errors similar to this one here: https://github.com/swordev/suid/issues/104 when trying to integrate SSR with existing app which uses SUID (I'm using vite-plugin-ssr for this). I had to write my own 'useMediaQuery' hook, because current one uses only window object, because of which I'm getting another errors. Is there any way to use SUID in SSR apps, or will this be possible in near future?

**Comment by juanrgm** [comment]
https://github.com/swordev/suid/issues/98#issuecomment-1272825196
